module/Frontpage/view/frontpage/frontpage/home.phtml

```diff
@@ -206,7 +206,7 @@ use Activity\Model\Activity;
                                     'h' => round($wThumbnailWidth * $wAspectRatio)
                                 ];
 
-                                $wLargeWidth = $photoConfig['large_thumb_size']['width'];
```

Comment:
Please do the same for the birthday photo.

Reply:
And the company banner perhaps?

Reply:
I have changed the birthday photo. Company banners are pretty small, so I don't think it will be necessary for now.